libs/docs/core/notification/examples/notification-mobile/notification-mobile-example.component.html

```diff
@@ -0,0 +1,263 @@
+<h4>Notification In Popover Dialog</h4>
+
+<button fd-button label="Notification" fdCompact [fdPopoverTrigger]="popoverMobile"></button>
```

Comment:
Some buttons are hardcoded compact and some respond to the content density observer. Is this intentional? Looks like in the specs they should all be compact

Reply:
Good catch, fixed for all examples